lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -205,10 +209,10 @@ func (ic *ICache) getImageByNamed(named reference.Named) *metadata.ImageConfig {
 // Don't assume the image id in image has "sha256:<id> as format.  We store it in
 // this format to make it easier to lookup by digest
 func prefixImageID(imageID string) string {
-	if strings.HasPrefix(imageID, "sha256:") {
+	if strings.HasPrefix(imageID, sha256Prefix) {
```

Comment:
If you make the change I suggest to avoid the `sha256` assumption, then this would change to `strings.Contains(k, ":")` instead of the prefix check.
In this case sha256 because the _default_ hash.

Reply:
Fixed!